examples/fishjam-chat/components/VideosGrid.tsx

```diff
@@ -55,6 +55,7 @@ export default function VideosGrid({ tracks }: Props) {
           <VideoRendererView
             trackId={track.id}
             videoLayout="FIT"
+            alwaysRender={false}
```

Comment:
**suggestion:** 
I think that this name can be confusing and could be renamed to something self explaining`shouldRenderIfViewIsNotAtScreen`, `renderOffscreen` or anything you find not confiusing.